x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlConfigMigrator.java

```diff
@@ -90,6 +94,7 @@
     private static final Logger logger = LogManager.getLogger(MlConfigMigrator.class);
 
     public static final String MIGRATED_FROM_VERSION = "migrated from version";
+    public static final int CONFIG_INDEX_MAX_RESULTS_WINDOW = 10_000;
```

Comment:
This setting is misplaced here. Could you move it to `AnomalyDetectorsIndex` please?

Reply:
Also, we should take the chance and raise this to 100K?